src/helpers/NodeHelper.ts

```diff
@@ -20,6 +20,12 @@ interface IInvalidNode {
  */
 export class NodeHelper {
   static _invalidNodesKey = "invalidNodes";
+  static _maxFailedConnections = 2;
+  /**
+   * failedConnectionsMinuteLimit is the number of minutes that _maxFailedConnections must occur within
+   * for the node to be blocked.
+   */
+  static _failedConnectionsMinutesLimit = 15;
```

Comment:
CodeStyle comment: when prefixing with `_`, you generally don't want others to access that method property/method. is there a reason why its prefixed with it, especially if you're using it elsewhere?